app/core/models/ocelot/stanzas/SequenceStanza.scala

```diff
@@ -52,7 +52,7 @@ object SequenceStanza {
     )(unlift(SequenceStanza.unapply))
 }
 
-trait Sequence extends VisualStanza with Populated with DataInput {
+trait SequenceLike extends VisualStanza with Populated with DataInput {
```

Comment:
SequenceLike no longer need, it content can be moved into the Sequence case class